Core/include/Acts/TrackFitting/Chi2FitterError.hpp

```diff
@@ -0,0 +1,28 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021-2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <system_error>
+
+namespace Acts {
```

Comment:
Actually, I would like to come back to the `Acts::Experimental` namespace. 

It would be a really easy change to just add `namespace Experimental` below here, but would have several advantages:
* It is clear that this code is not yet production ready. We are a Open-source library and have several clients, and if they see this in the release notes, some of them might try this out, which is now maybe not yet ready for it.
* Without it is part of the public API. That means, with every change to the interface, the options, the return type we have a breaking change for that we need to bump the major version of whole ACTS. I think this is not good for a not yet polished component.

I promise you its a really easy change :)

Reply:
Since in the meantime PR #1609 got merged, I will also include that change here.

@benjaminhuth  could you have a look, if I implemented the changes correctly?

Reply:
Looks good to me!